src/test/python/fusionauth/fusionauth_client_test.py

```diff
@@ -84,6 +84,25 @@ def test_create_user_retrieve_user(self):
         self.assertFalse('password' in get_user_response.success_response['user'])
         self.assertFalse('salt' in get_user_response.success_response['user'])
 
+        # Retrieve the user via loginId
+        get_user_response = self.client.retrieve_user_by_login_id('user@example.com')
+        self.assertEqual(get_user_response.status, 200)
+        self.assertIsNotNone(get_user_response.success_response)
+        self.assertIsNone(get_user_response.error_response)
+        self.assertEqual(get_user_response.success_response['user']['email'], 'user@example.com')
+
+        # Explicit loginIdType
+        get_user_response = self.client.retrieve_user_by_login_id('user@example.com', ['email'])
+        self.assertEqual(get_user_response.status, 200)
+        self.assertIsNotNone(get_user_response.success_response)
+        self.assertIsNone(get_user_response.error_response)
+        self.assertEqual(get_user_response.success_response['user']['email'], 'user@example.com')
+
+# TODO: Once issue 1 is released, this test should pass
```

Comment:
Since we don't have a releaseable docker image yet, this has to stay commented out, but passes locally with a dev image